internal/eventlog/metrics.go

```diff
@@ -37,3 +37,9 @@ func PrometheusMetrics(ns string, fields ...string) *Metrics {
 		}, labels).With(fields...),
 	}
 }
+
+func NopMetrics() *Metrics {
```

Comment:
The other `NopMetrics` seem to be generated code.
Is there a reason this code is manual?

Reply:
Good catch, since other metrics [generated](https://github.com/tendermint/tendermint/pull/8488/files) after evetlog get introduced.

Reply:
Thanks a lot, @mmsqe !